### ISSUE TITLE
App hangs/crashes on table refresh when table is no longer available

### ISSUE THREAD

**Issue by EdwinMeijne** [sequelpro/sequelpro#2146]
Hi

I am working with PHP/Laravel in a homestead virtual machine (vagrant), pretty standard. I use the migrations in Laravel and often 'refresh' the migrations as I am fiddling with the table names and structure. When I use Sequel through a local connection to the VM, nothing special and never gives issues.
![screenshot 2015-06-17 12 28 47](https://cloud.githubusercontent.com/assets/7081420/8204903/70328842-14ec-11e5-83b0-c54cc98f27e8.png)

When I have a table content view open (I something manually change values in the database for testing), but when I 'refresh' the migrations (delete all tables and re-create/populate tables) the program hangs when I accidentally refresh the content-view that was still open.

So, TL;DR:
Sequel crashes when trying to refresh a table that doesn't exist anymore (and is recreated with the same name)

**Comment by EdwinMeijne** [comment]
*screenshot of 'content' view when bug happens
![screenshot 2015-06-17 12 31 55](https://cloud.githubusercontent.com/assets/7081420/8204988/e39b64a2-14ec-11e5-9e3b-420ace657730.png)

**Comment by dmoagx** [comment]
Have your tried 1.1rc1?
http://www.sequelpro.com/blog/2015.06/sequel-pro-1-1-rc1/

**Comment by EdwinMeijne** [comment]
Hi

No, not yet. I am using the latest non-beta version. Should the rc1 fix the issue?
Now I have to choose between a beta release with maybe some unknown bugs or a stable version with a single bug that I know how (not) to trigger... hmm...

**Comment by EdwinMeijne** [comment]
As I am using Sequel Pro in a please-dont-fail-during-my-work-on-this-clients-server environment I will stick with the stable release and update as soon as the RC becomes the new release; I did however made a donation and would like to thank all the developers out here for their hard work, you made my life easier ever since I discovered this program!

**Comment by dmoagx** [comment]
Closing after no feedback from user.
